integration/teleterm_test.go

```diff
@@ -335,6 +342,77 @@ func testHeadlessWatcher(t *testing.T, pack *dbhelpers.DatabasePack, creds *help
 	)
 }
 
+func testRemoteClientCache(t *testing.T, pack *dbhelpers.DatabasePack, creds *helpers.UserCreds) {
+	t.Helper()
```

Comment:
Why is the whole test marked as a helper function?

Reply:
I think I copied it from an other test, removed.